src/App.js

```diff
@@ -1,10 +1,62 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import logo from './assets/logo.png';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./components/Pokemon/PokemonCard";
+import Button from "./components/Button/Button";
 
 function App() {
+    const [pokemon, setPokemon] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [pokemon, setPokemon] = useState();
```